backend/tests/reservation/getAllForRestaurant.js

```diff
@@ -41,19 +39,19 @@ describe('GET reservations/', () => {
     // assert.isObject(body, 'Expected response to contain a body object');
     // assert.isArray(body.result, 'Expected response body to contain an array of results');
     // assert.deepEqual(body.result, [], 'Expected an empty array of results');
-    assert.isOk(true)
+    assert.isOk(true);
   });
 
   // Test that the API is able to return the correct result when there is only one reservation for the desired restaurant.
   it('2. should return an array of reservations when the restaurant only has one reservation', async function () {
-    // await connection
-    //   .asyncQuery(
-    //     `INSERT INTO RESERVATION (ID, Date, Time, Notes, NumberOfGuests, TableID, RestaurantID, UserID)
-    // VALUES (1, '2020-03-14', '11:30:00', '', '1', '1', '1', '1');`
-    //   )
-    //   .then(({ error }) => {
-    //     assert.isUndefined(error, 'Expected no errors when adding items to database');
-    //   });
+    await connection
+      .asyncQuery(
+        `INSERT INTO restaurant_db.RESERVATION (ID, Date, Time, Notes, NumberOfGuests, TableID, RestaurantID, UserID)
```

Comment:
I think 'restaurant_db.' was removed so different database names could be used. Could you please check and remove these from the other test cases too (other test files as well)? Thanks